internal/sbi/api_accesstoken.go

```diff
@@ -77,7 +98,9 @@ func HTTPAccessTokenRequest(c *gin.Context) {
 		}
 	}
 
-	err = c.Bind(&accessTokenReq)
+	if err1 := s.bindData(c, &accessTokenReq); err1 != nil {
```

Comment:
err1 should be err ?

Reply:
No, the err1 is different from err. 
The err was previously declared and initialized to store the error value returned by the r.ParseForm() (see in line 64)